flink-runtime-web/web-dashboard/app/partials/jobs/job.exceptions.jade

```diff
@@ -31,7 +35,9 @@
 
   .panel-heading.clearfix
     .panel-info.thin.last
-      span {{ exception.location }}
+      | Timestamp: {{ exceptions['timestamp'] | amDateFormat:'YYYY-MM-DD, H:mm:ss' }}
+      br
+      | Location: {{ exception.location }}
```

Comment:
I would do multiple panels like this:
```
.panel-info.thin
  strong Timestamp:
  = ' '
  | {{ exceptions['timestamp'] | amDateFormat:'YYYY-MM-DD, H:mm:ss' }}
.panel-info.thin.last
  strong Location:
  = ' '
  | {{ exception.location }}
```

![screen shot 2017-03-21 at 11 20 06](https://cloud.githubusercontent.com/assets/1756620/24143019/a6105548-0e28-11e7-9d35-5522c7626845.png)